src/app/(BottomTabNavigation)/AllCases/utils.ts

```diff
@@ -36,8 +36,23 @@ export async function fetchAllCases(userUid: UserUid): Promise<Case[]> {
  * @returns readable date string
  */
 export function formatDate(dateObject: Date) {
-  const date = new Date(dateObject).toDateString().split(' ');
-  return `${date[1]} ${date[2]}, ${date[3]}`;
+  const date = new Date(dateObject);
+  const months = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ];
+  const monthName = months[date.getUTCMonth()];
+  return `${monthName} ${date.getUTCDate()}, ${date.getUTCFullYear()}`;
 }
 
 export function getStatusColor(status: string) {
```

Comment:
I have an idea for the `getStatusColor` function. We should create types for each status color type, such as `GreenStatusOptions`, and set it equal to the union of all options: `'In Progress' | 'New Case' | ...`. Do so for each of the four color options. Then union these options to create `CaseStatusOptions` type. That way if we want to adjust status options, we only have to do it in one place.